fbw-a380x/src/systems/systems-host/systems/FlightWarningSystem/FwsAbnormalSensed.ts

```diff
@@ -3284,7 +3330,7 @@ export class FwsAbnormalSensed {
     },
     320800018: {
       // BRAKES HOT
-      flightPhaseInhib: [4, 5, 9, 10, 11, 12],
+      flightPhaseInhib: [4, 5, 9, 10],
```

Comment:
```suggestion
      flightPhaseInhib: [4, 5, 9, 10, 11],
```
my batch 7 ref also says flight phase 11